examples/standalone_single_project/cicd.tf

```diff
@@ -66,12 +66,13 @@ module "ci_pipeline" {
   cloudbuild_private_pool   = module.cloudbuild_private_pool.workerpool_id
   clouddeploy_pipeline_name = local.clouddeploy_pipeline_name
   skip_provisioners         = true
+  labels                    = var.labels
```

Comment:
For catching issues like this in the future where a var was defined but not used, you can bump devtools to 1.9 or higher which has tflint rules which catch these.